parser/src/main/scala/com/intuit/superglue/pipeline/parsers/CalciteStatementParser.scala

```diff
@@ -12,18 +13,33 @@ import org.apache.calcite.sql.validate.SqlConformanceEnum
 import scala.util.{Failure, Success, Try}
 
 class CalciteStatementParser extends StatementParser {
-  private val calciteConfig: SqlParser.Config = SqlParser.configBuilder()
-    .setParserFactory(SqlDdlParserImpl.FACTORY)
-    .setConformance(SqlConformanceEnum.MYSQL_5)
-    .build()
+  private val defaultCalciteConfig: SqlParser.Config = SqlParser.config()
+    .withParserFactory(SqlDdlParserImpl.FACTORY)
+    .withConformance(SqlConformanceEnum.MYSQL_5)
```

Comment:
Change this to conformance level - BABEL. This is more liberal and allows queries supported by any dialect.